src/components/Hub/ImageCard.tsx

```diff
@@ -43,10 +43,11 @@ const ImageLink = styled(Link)`
 `
 
 export default function ImageCard({ image, index }: Props) {
-  let { name, author, keywords, description } = image
+  let { name, author, keywords, description } = image;
+  let url = btoa('/package/' + index);
```

Comment:
@pdaryamane, could you explain what problem you're trying to solve here?

Reply:
@Roshanjossey : In JINA Dashboard UI, the url is given as /package/<number_of_the_package> when we select a new package to see the details. Instead of showing the actual package name, the encoded url will now be shown. Used windows base encoding to do this.

Reply:
Oh, I see the confusion here. The issue doesn't mean encode the URL, on the contrary, it's to make the URLs more easy to read, share and discover.

We have different image types like `encoder`, `crafter`, `ranker` etc. The suggestion in the pull request is to change the URLs to have that information in the URL too.

Reply:
ok, thanks for reviewing. Can you please let me know where can I find all the image types, so that I can check if I can work on the issue

Reply:
Sure, the following are the image types we have at the moment

```
classifier
crafter
encoder
evaluator
indexer
ranker
segmenter
```

You can get details of images with Request URL: https://hubapi.jina.ai/images
`kind` attribute of images is type.